src/dmd/root/ctfloat.h

```diff
@@ -59,6 +59,4 @@ struct CTFloat
     static real_t one;
```

Comment:
Not sure if they are used, but these statics no longer exist unconditionally.

Reply:
How do I handle that? Perhaps better to now use aliases?

Reply:
> How do I handle that? Perhaps better to now use aliases?

s/now/not/? Yes, probably simpler to just make a copy for the symbols as the non-LDC version.